src/ObjectPool/src/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,12 @@
 #nullable enable
+Microsoft.Extensions.DependencyInjection.ObjectPoolServiceCollectionExtensions
 Microsoft.Extensions.ObjectPool.IResettable
 Microsoft.Extensions.ObjectPool.IResettable.TryReset() -> bool
+Microsoft.Extensions.ObjectPool.PoolOptions
+Microsoft.Extensions.ObjectPool.PoolOptions.PoolOptions() -> void
+Microsoft.Extensions.ObjectPool.PoolOptions.Capacity.get -> int
+Microsoft.Extensions.ObjectPool.PoolOptions.Capacity.set -> void
+static Microsoft.Extensions.DependencyInjection.ObjectPoolServiceCollectionExtensions.AddPooled<TService, TImplementation>(this Microsoft.Extensions.DependencyInjection.IServiceCollection! services, System.Action<Microsoft.Extensions.ObjectPool.PoolOptions!>? configureOptions = null) -> Microsoft.Extensions.DependencyInjection.IServiceCollection!
+static Microsoft.Extensions.DependencyInjection.ObjectPoolServiceCollectionExtensions.AddPooled<TService>(this Microsoft.Extensions.DependencyInjection.IServiceCollection! services, System.Action<Microsoft.Extensions.ObjectPool.PoolOptions!>? configureOptions = null) -> Microsoft.Extensions.DependencyInjection.IServiceCollection!
+static Microsoft.Extensions.DependencyInjection.ObjectPoolServiceCollectionExtensions.ConfigurePool<TService>(this Microsoft.Extensions.DependencyInjection.IServiceCollection! services, System.Action<Microsoft.Extensions.ObjectPool.PoolOptions!>! configureOptions) -> Microsoft.Extensions.DependencyInjection.IServiceCollection!
+static Microsoft.Extensions.DependencyInjection.ObjectPoolServiceCollectionExtensions.ConfigurePools(this Microsoft.Extensions.DependencyInjection.IServiceCollection! services, System.Action<Microsoft.Extensions.ObjectPool.PoolOptions!>! configureOptions) -> Microsoft.Extensions.DependencyInjection.IServiceCollection!
```

Comment:
This method wasn't approved in API review.

Reply:
It was is this file but not in the code. Fixed.